### PR TITLE
upload team logos to new location for panorama

### PR DESCRIPTION
**Pull request by pikaxo**
as title said its just a correction because panorama

### CODE REVIEW

**Comment by splewis** [comment]
Can you make this change backwards compatible? I'm willing to change the default to svg and the new paths, but not drop support for existing behavior entirely. A new cvar would be fine.

File: scripting/get5_apistats.sp [hunk 1]
```diff
@@ -35,14 +35,20 @@
...
 char g_APIKey[128];
 
 ConVar g_APIURLCvar;
 char g_APIURL[128];
 
-#define LOGO_DIR "resource/flash/econ/tournaments/teams"
+//This logo dir its for old csgo hud
+    if (g_UseSVGCvar.BoolValue) {
+      #define LOGO_DIR "materials/panorama/images/tournaments/teams"
+    } else {
```
Comment (splewis) [inline scripting/get5_apistats.sp]: This doesn't work  (or compile), can you do something like:

```
#define LEGACY_LOGO_DIR  "resource/flash/econ/tournaments/teams"
#define LOGO_DIR  "materials/panorama/images/tournaments/teams"
```

Then you can do

```
  if (g_UseSVGCvar.BoolValue) {
    Format(logoPath, sizeof(logoPath), "%s/%s.svg", LOGO_DIR, logo);
  } else {
    Format(logoPath, sizeof(logoPath), "%s/%s.png", LEGACY_LOGO_DIR , logo);
  }
```

Comment (pikaxo) [reply scripting/get5_apistats.sp]: i saw it on same day but it's too late for me

**Comment by splewis** [comment]
Was added in #498

**Pull request closed without merging**